src/main/java/net/smartcosmos/cluster/userdetails/DevelopmentUserDetailsService.java

```diff
@@ -1,13 +1,15 @@
 package net.smartcosmos.cluster.userdetails;
 
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Import;
 
-import net.smartcosmos.cluster.userdetails.config.UserAuthenticationProperties;
+import net.smartcosmos.cluster.userdetails.config.UserDetailsDevelopmentConfiguration;
 
 @SpringBootApplication
-@EnableConfigurationProperties({ UserAuthenticationProperties.class })
+@EnableAutoConfiguration
+@Import(UserDetailsDevelopmentConfiguration.class)
```

Comment:
I've been having problems with autowiring.  I am trying to get tests going that do not require loading the entire works and all the web mvn components.  It's still not correct